src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -433,8 +433,17 @@ Whether Harvesting (OAI) service is enabled
          * Installation Brand Name is always included (default/false) or is not included
          * when the Distributor field (citation metadatablock) is set (true)
          */
-        ExportInstallationAsDistributorOnlyWhenNotSet
-        ;
+        ExportInstallationAsDistributorOnlyWhenNotSet,
+
+        /**
+         * A Json object containing key/values corresponding the the allowed languages
+         * for entering metadata with the key being the human-readable string and the
+         * value being an ISO-639 code for that language (2 and 3 letter codes from the
+         * 639-2 and 639-3 standards are allowed. These will be used directly in
+         * metadata exports.) These values are selectable at the Dataverse level and
+         * apply to Dataset metadata.
+         */
+        MetadataLanguages;
```

Comment:
would it possibly make sense to not have two sets of languages - one for interface and one for metadata and expect installation to use same languages for both? I guess there could be a use case for different, but I wonder if it's small enough that the more simple approach (one shared set) isn't more beneficial?

Reply:
Sciences PO has use cases where these will be different. (My understanding is that they may only support a couple :Languages and allow more :MetadataLanguages, but I could image others wanting the reverse (support lots of display languages but restrict entries to a few.))